### PR TITLE
Correct flake8 checkstyle issues

### PR DESCRIPTION
**Pull request by patcon**
Reference #152

### CODE REVIEW

**Comment by jpmckinney** [comment]
Hmm, some of these changes to satisfy flake8 should not have been performed (notably the "line too long") issues. Did you use autopep8? You need to run it with `--ignore E501,E731` to match what's in tox.ini

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+language: python
+
```
Comment (jpmckinney) [inline .travis.yml]: Remove extra newlines between keys

**Comment by patcon** [comment]
Fixed with history rewrite.

**Comment by jpmckinney** [comment]
:100: I enabled Travis now.

**Comment by jpmckinney** [comment]
Just noticed that the Travis badge was lost after rewrite.

**Pull request merged**